tests/test_products.py

```diff
@@ -77,10 +77,8 @@ def test_products(requests_mock):
     assert len(openshift410z.yum_repositories) == 5
 
     rhacm24z = ProductStream.objects.get(name="rhacm-2.4.z")
-    assert et_variant.name in rhacm24z.productvariants.values_list("name", flat=True)
-    # Stream "cpes" is a dynamically-generated property, different than the "cpe" field
-    # Which should include the stream's CPE + all the child variant CPEs (if any)
-    assert et_variant.cpe in rhacm24z.cpes
+    assert et_variant.name not in rhacm24z.productvariants.values_list("name", flat=True)
+    assert et_variant.cpe in rhacm24z.cpes_from_brew_tag_variants
```

Comment:
If there's no longer a child Variant for this Brew tag, let's `assert et_variant.cpe not in rhacm24z.cpes`, so that we know the "cpes" property is only including CPEs from linked Variants, and not including cpes from any of the fields like "cpes_matching_patterns" or "cpes_from_brew_tag_variants".
```suggestion
    assert et_variant.cpe not in rhacm24z.cpes
    assert et_variant.cpe in rhacm24z.cpes_from_brew_tag_variants
```

It's also a little minor, but maybe we should call the new field "cpes_from_brew_tag_matches" or just "cpes_from_brew_tags", since the brew_tag_variants don't really exist anymore.

Reply:
Good idea.